bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/protocol/leap/LeapMessageParser.java

```diff
@@ -181,6 +182,11 @@ private void handleReadResponseMessage(JsonObject message) {
                 case "OneZoneStatus":
                     parseOneZoneStatus(body);
                     break;
+                case "OneProjectDefinition":
+                    parseOneProjectDefinition(body);
+                    break;
+                case "OneDeviceDefinition":
+                    parseOneDeviceDefinition(body);
```

Comment:
Please confirm that the miss of a break here is really expected.

Reply:
great catch, it needs it